pkg/util/file/file.go

```diff
@@ -174,15 +175,24 @@ func WriteIdentity(idFile string, idEntry identity.RekorLogEntry) error {
 
 // WriteIdentityMetadata writes information about what log indices have been scanned to a file
 func WriteIdentityMetadata(metadataFile string, idMetadata IdentityMetadata) error {
-	file, err := os.OpenFile(metadataFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err := os.WriteFile(metadataFile, []byte(idMetadata.String()), 0600); err != nil {
```

Comment:
Can we have `IdentityMetadata` be a JSON structure instead of a string? This would make it easier to extend the metadata later on. We need to add a JSON tag to the field in `IdentityMetadata`, marshal it before writing, and unmarshal on read.